migration-engine/migration-engine-tests/src/assertions.rs

```diff
@@ -71,6 +71,7 @@ impl SchemaAssertion {
         }
     }
 
+    #[track_caller]
```

Comment:
This puts the error on the right line in the test, instead of inside the assertion function.